.ci/Jenkinsfile

```
@@ -536,6 +502,11 @@ def generateFunctionalTestStep(Map args = [:]){
     goArch = "arm64"
   }
 
+  // sanitize tags to create the file
+  def sanitisedTags = tags.replaceAll("\\s","_")
+  sanitisedTags = sanitisedTags.replaceAll("~","")
+  sanitisedTags = sanitisedTags.replaceAll("@","")
+
   def githubCheckSha1 = params.GITHUB_CHECK_SHA1?.trim() ? params.GITHUB_CHECK_SHA1 : ''
   def githubCheckRepo = params.GITHUB_CHECK_REPO?.trim() ? params.GITHUB_CHECK_REPO : ''
```

IIUC, those env variables are already defined in the top-level environment section with `trim()`:

```suggestion
  def githubCheckSha1 = env.GITHUB_CHECK_SHA1
  def githubCheckRepo = env.GITHUB_CHECK_REPO
```

---

Yes, I saw that. But I'm consistently seeing a `"null"` value being passed to Ansible: `GithubCommitSha=null`

>[2022-07-04T05:35:35.576Z] time="2022-07-04T05:35:34Z" level=info msg="Initial artifact versions defined" BeatVersion=7.17.5-b5794350-SNAPSHOT BeatVersionBase=7.17.5-b5794350-SNAPSHOT ElasticAgentVersion=7.17.5-b5794350-SNAPSHOT GithubCommitSha=null GithubRepository=elastic-agent KibanaVersion=7.17.5-b5794350-SNAPSHOT StackVersion=7.17.5-b5794350-SNAPSHOT

URL: https://beats-ci.elastic.co/blue/organizations/jenkins/e2e-tests%2Fe2e-testing-mbp%2F7.17/detail/7.17/818/pipeline